diff --git a/packages/@tailwindcss-standalone/src/index.ts b/packages/@tailwindcss-standalone/src/index.ts
index be85729b..756117da 100644
--- a/packages/@tailwindcss-standalone/src/index.ts
+++ b/packages/@tailwindcss-standalone/src/index.ts
@@ -15,6 +15,7 @@ globalThis.__tw_resolve = (id, baseDir) => {
     id === 'tailwindcss' ||
     id.startsWith('tailwindcss/') ||
     id.startsWith('@tailwindcss/') ||
+    id.startsWith('daisyui') ||
     isEmbeddedFileBase

   if (!likelyEmbeddedFile) {
@@ -44,6 +45,8 @@ globalThis.__tw_resolve = (id, baseDir) => {
     case '@tailwindcss/forms':
     case '@tailwindcss/typography':
     case '@tailwindcss/aspect-ratio':
+    case 'daisyui':
+    case 'daisyui/theme':
       return id
     default:
       return false
@@ -56,6 +59,10 @@ globalThis.__tw_load = async (id) => {
     return require('@tailwindcss/typography')
   } else if (id.endsWith('@tailwindcss/aspect-ratio')) {
     return require('@tailwindcss/aspect-ratio')
+  } else if (id.endsWith('daisyui')) {
+    return require('daisyui')
+  } else if (id.endsWith('daisyui/theme')) {
+    return require('daisyui/theme')
   } else {
     return undefined
   }
